tests/product/test_configuration.py

```diff
@@ -195,31 +195,31 @@ def test_configuration_deploy_using_dash_x_coord_worker(self):
                                      self.default_workers_test_config_)
 
     def test_lost_coordinator_connection(self):
-        internal_bad_host = self.cluster.internal_slaves[0]
-        bad_host = self.cluster.slaves[0]
+        bad_host = self.cluster.get_down_hostname()
         good_hosts = [self.cluster.internal_master,
                       self.cluster.internal_slaves[1],
                       self.cluster.internal_slaves[2]]
-        topology = {'coordinator': internal_bad_host,
+        topology = {'coordinator': bad_host,
                     'workers': good_hosts}
         self.upload_topology(topology)
-        self.cluster.stop_host(bad_host)
+        self.cluster.stop_host(self.cluster.slaves[0])
```

Comment:
you've already put the bad host name in the topology, so stopping the host should really do anything. Maybe we don't need stop_host at all, just upload bad worker/coordinator topologies all around.   In any case it should be one or the other.